### PR TITLE
[FIX] fix-get-default-company

### PR DESCRIPTION
**Pull request by cialuo**
Fix get default company
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/xie8899/project/pull/34%23issuecomment-197275852%22%2C%20%22https%3A//github.com/xie8899/project/pull/34%23discussion_r56464308%22%2C%20%22https%3A//github.com/xie8899/project/pull/34%23issuecomment-197744089%22%2C%20%22https%3A//github.com/xie8899/project/pull/34%23issuecomment-198167111%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/34%23issuecomment-197275852%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%27m%20not%20sure%20that%20the%20error%20related%20to%20this.%20on%20my%20local%20compute%2C%20i%20have%20not%20get%20the%20issue.%20i%20remove%20the%20company_id%20in%20business%2C%20it%20also%20have%20not%20the%20error.%22%2C%20%22created_at%22%3A%20%222016-03-16T11%3A32%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7767792%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cialuo%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-03-17T07%3A50%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-03-18T02%3A22%3A48Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12975184%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/siyuancn%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%204064e5b7da697a8a42c93fef4ba915128c82eeb5%20business_requirement/models/business.py%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/34%23discussion_r56464308%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22try%20this%20%3F%5Cr%5Cn%60%60%60python%5Cr%5Cn%40api.returns%28%27self%27%29%5Cr%5Cn%60%60%60%22%2C%20%22created_at%22%3A%20%222016-03-17T07%3A23%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20business_requirement/models/business.py%3AL16-28%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/34#issuecomment-197275852'>General Comment</a></b>
- <a href='https://github.com/cialuo'></a> I'm not sure that the error related to this. on my local compute, i have not get the issue. i remove the company_id in business, it also have not the error.
- <a href='https://github.com/duanyp1991'></a> LGTM
- <a href='https://github.com/siyuancn'></a> LGTM
- [x] <a href='#crh-comment-Pull 4064e5b7da697a8a42c93fef4ba915128c82eeb5 business_requirement/models/business.py 3'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/34#discussion_r56464308'>File: business_requirement/models/business.py:L16-28</a></b>
- <a href='https://github.com/duanyp1991'></a> try this ?

``` python
@api.returns('self')
```

<a href='https://www.codereviewhub.com/xie8899/project/pull/34?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/xie8899/project/pull/34?approve=1'></a>&nbsp;<a href='https://github.com/xie8899/project/pull/34'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by xie8899** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/xie8899/project/pull/34'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by cialuo** [comment]
I'm not sure that the error related to this. on my local compute, i have not get the issue. i remove the company_id in business, it also have not the error.

File: business_requirement/models/business.py [hunk 1]
```diff
@@ -16,12 +16,12 @@ class BusinessRequirement(models.Model):
 
     @api.model
     def _get_default_company(self):
```
Comment (duanyp1991) [inline business_requirement/models/business.py]: try this ?
```python
@api.returns('self')
```

**Comment by duanyp1991** [comment]
LGTM

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**